### PR TITLE
Update teleport command descriptions for consistency

### PR DESCRIPTION
**Pull request by stevenGravy**
Commands should end with a `.`

### CODE REVIEW

File: e
Comment (zmb3) [inline e:1]: Did you intend to bundle an e update in here?

**Review by zmb3 — APPROVED** [review]

File: e
Comment (stevenGravy) [reply e:1]: I didn't think we need to since this is in `teleport` repo.  If there's a step i'm missing here happy to

**Review by scottpgallagher — APPROVED** [review]
Approved.

File: e
Comment (stevenGravy) [reply e:1]: ah, my mistake, will fix

**Pull request merged**